### DISCUSSION TITLE
Disable Execptions

### DISCUSSION THREAD

**Discussion by nikuscs** [spatie/ray#350] [Ideas]
How do we disable Execptions reporting to ray? In my opinion it creates a lot of uncessary "noise" while debugging. Iv tried to read the docs but could not find anything relevant to turn it off.

**Comment by freekmurze** [comment]
I think we should add this as a config option, I'd accept a PR that adds this. If none comes in, I'll try to add it soon-ish

**Reply by patinthehat** [reply]
@freekmurze @igerslike 
Are we talking about disabling the `exception()` method on the `Ray` class via a config option?

**Reply by nikuscs** [reply]
@patinthehat I'm not sure if that's what you mean, but my suggestion  would be disable all **automatic** exceptions reporting via config file. Currently all exceptions are being logged to Ray which is fine in most of the cases, but some cases i believe it creates a lot of "trash" on the UI. I believe if you still call the exception() method they should be still logged as you intentionally want to see it. This could also result in some command like `ray()->dontReportExceptions()`. I will try and see if i can push a PR later today.

**Reply by patinthehat** [reply]
@igerslike Oh, I think you're referring to `spatie/laravel-ray` and not `spatie/ray`?

**Reply by nikuscs** [reply]
@patinthehat I'm not sure where this could apply to be honest, im not entirely aware of the project structure, but this could be applied to all adaptors that supports exceptions logging, if you can point me in the right direction i could probably start digging on it.

**Reply by patinthehat** [reply]
I managed to find the code in question - it is in the `laravel-ray` package.  I will endeavor to submit a PR for this feature some time today.

**Reply by patinthehat** [reply]
@igerslike I've submitted a [PR to spatie/laravel-ray](https://github.com/spatie/laravel-ray/pull/173) for this feature.

**Reply by nikuscs** [reply]
@patinthehat thanks a lot mate! Just watching the PR right now!

**Reply by freekmurze** [reply]
merged and released, thanks @patinthehat 👍
